mm2src/coins/lp_coins.rs

```diff
@@ -377,6 +377,7 @@ pub enum TransactionErr {
     TxRecoverable(TransactionEnum, String),
     /// Simply for plain error messages.
     Plain(String),
+    InvalidTx(String),
```

Comment:
I have two suggestions:
1) I think we need to divide `TransactionErr` into `GenSendTxError` and `ParseTxError`.
2) It would be better to add `GenSendTxErrorKind` and refactor `GenSendTxError` this way:
```rust
pub enum GenSendTxError {
  Recoverable(TransactionEnum, GenSendTxErrorKind),
  Plain(GenSendTxErrorKind),
}
```
Otherwise it may confuse if you get `TransactionErr::TxRecoverable` on `coin.tx_enum_from_bytes`.

Reply:
I agree with using different error structure. I will create new a error enum for this specific use case.

Do you think it's I should refactor `TransactionErr` in this PR? I think it's more relevant with #1433. Please let me know if I should do it here.

Reply:
Since `send_raw_tx` and `send_raw_tx_bytes` will be refactored on #1433 it can directly use the refactored version of `TransactionErr`.

I also added a TODO as a reference.
<https://github.com/KomodoPlatform/atomicDEX-API/blob/4ae28fbe7df672bd522c558a14cf1076cb69df61/mm2src/mm2_main/src/lp_network.rs#L170-L172>


PS: Specific error type is created and used instead `TransactionErr`.

Reply:
> Do you think I should refactor TransactionErr in this PR? I think it's more relevant with https://github.com/KomodoPlatform/atomicDEX-API/pull/1433. Please let me know if I should do it here.

Let's refactor it in #1443

Reply:
> I agree with using different error structure. I will create new a error enum for this specific use case.

Great, thank you